ioc/ioc_test.go

```diff
@@ -182,11 +188,15 @@ func TestGetIOCs(t *testing.T) {
 	}
 
 	for _, test := range testsStandardizedDefangs {
-		if iocs := GetIOCs(test.input, true, true); !reflect.DeepEqual(iocs, test.want) {
-			t.Errorf("[standardizedDefang=true] IOCType(%q), found %v =/= wanted %v", test.input, iocs, test.want)
-		}
-	}
+		t.Run(test.input, func(t *testing.T) {
```

Comment:
So what's the advantage of doing this `t.Run` format instead of the way I did it?

Reply:
Does it break out the tests more in the output or something?

Reply:
Yes, it does. When a test fails you see the name (here test.input) of the case that failed. Try breaking a test and see for yourself. Most IDEs support it as well (intelliJ in my case)